doc/rvv-intrinsic-spec.adoc

```diff
@@ -560,66 +560,85 @@ Availability of the tuple types follows the availability of their corresponding
 [[pseudo-intrinsics]]
 == Pseudo intrinsics
 
-The intrinsics provide additional utility functions to assist users in manipulating across intrinsic types. These functions are called "pseudo intrinsics". These pseudo intrinsics do not represent any real instructions.
+Pseudo intrinsics provide additional utility functions to assist users in manipulating across intrinsic types. They do not map to any specific RVV instruction. The specific mapping to actual instructions is given in the description of each pseudo intrinsic.
 
 [[pseudo-vsetvl]]
 === `vsetvl`
 
-The `vsetvl` intrinsics return the number of elements processed in a stripmining loop when  provided with the element width and LMUL in the intrinsic suffix.
+The `vsetvl` intrinsics return the number of elements processed in a stripmining loop when  provided with the element width and LMUL in the intrinsic suffix. This pseudo intrinsic is typically mapped to `vsetvli` or `vsetivli` instructions.
 
-NOTE: The implementation must respect the ratio between SEW and LMUL given to the intrinsic. On the other hand, as mentioned in <<control-of-vl>>, the `vsetvl` intrinsics do not necessarily map to the emission a `vsetvl` instruction of that exact SEW and LMUL provided. The actual value written to the `vl` control register is an implementation defined behavior and typically not known until runtime.
+NOTE: The implementation must respect the ratio between SEW and LMUL given to the intrinsic. On the other hand, as mentioned in <<control-of-vl>>, the `vsetvl` intrinsics do not necessarily map to the emission a `vsetvli` or `vsetivli` instruction of that exact SEW and LMUL provided. The actual value written to the `vl` control register is an implementation defined behavior and typically not known until runtime.
```

Comment:
Are we missing `vsetvl` instruction?

Reply:
The omission is intentional: `vtype` is statically encoded in the API.